src/js/tracing/integrations/appStart.ts

```diff
@@ -123,11 +123,10 @@ export const appStartIntegration = ({
     }
   };
 
-  const afterAllSetup = (client: Client): void => {
+  const afterAllSetup = (_client: Client): void => {
     if (standaloneUserOption === undefined) {
       // If not user defined, set based on the routing instrumentation presence
-      standalone = !client.getIntegrationByName<ReactNativeTracing>(ReactNativeTracing.id)?.options
-        .routingInstrumentation;
+      // FIXME: standalone = getReactNativeTracingIntegration(client)?.options.routingInstrumentation;
```

Comment:
Q: Are you gonna fix it on a follow up PR?

Reply:
Thank you for the catch, this will likely change in the future, but for now I left it as is, I removed the comment.